app/lib/handover/handover_email.rb

```diff
@@ -9,7 +9,9 @@ def self.deliver_if_deliverable(handover_email_type, nomis_offender_id, staff_me
       return
     end
 
-    mailer = HandoverMailer.public_send(handover_email_type, **mailer_args.merge(nomis_offender_id: nomis_offender_id))
+    # mailer = HandoverMailer.public_send(handover_email_type, **mailer_args.merge(nomis_offender_id: nomis_offender_id))
```

Comment:
I assume this has just been missed and can be removed? If it's still required and/or being kept for posterity, please ignore me!

Reply:
Fiaxed